components/org.wso2.carbon.identity.api.server.configs/org.wso2.carbon.identity.api.server.configs.v1/src/main/java/org/wso2/carbon/identity/api/server/configs/v1/core/ServerConfigManagementService.java

```diff
@@ -482,6 +483,7 @@ public void resetRemoteServerConfig(String logType) {
 
         RemoteServerLoggerData remoteServerLoggerData = new RemoteServerLoggerData();
 
+        logType = logType.toUpperCase(Locale.ENGLISH);
```

Comment:
This not prevent from using /AUDIT or /CARBON right?

Reply:
As per the discussion with @madurangasiriwardena it is decided to allow only `audit` and `carbon` endpoint without allowing case insensitive endpoints.
Hence this is fixed with https://github.com/wso2/identity-api-server/pull/573/commits/153528ed991d2db1be1991fb740d9d22bcf2687f